io/module_wrt_grid_comp.F90

```diff
@@ -469,8 +470,31 @@ subroutine wrt_initialize(wrt_comp, imp_state_write, exp_state_write, clock, rc)
         call ESMF_Finalize(endflag=ESMF_END_ABORT)
 
       endif
+!
+!-----------------------------------------------------------------------
+!***  get write grid component initial time from clock
+!-----------------------------------------------------------------------
+!
+      call ESMF_ClockGet(clock    =CLOCK                                &  !<-- The ESMF Clock
+                        ,startTime=wrt_int_state%IO_BASETIME            &  !<-- The Clock's starting time
+                        ,rc       =RC)
 
-
+      call ESMF_TimeGet(time=wrt_int_state%IO_BASETIME,yy=idate(1),mm=idate(2),dd=idate(3),h=idate(4), &
+                        m=idate(5),s=idate(6),rc=rc)
+!     if (lprnt) write(0,*) 'in wrt initial, io_baseline time=',idate,'rc=',rc
+      idate(7) = 1
+      wrt_int_state%idate = idate
+      wrt_int_state%fdate = idate
```

Comment:
add a comment here since it's not obvious what's going on. Something like "update idate  IAU is enabled"

Reply:
Added comments "! update IO-BASETIME and idate on write grid comp when IAU is enabled"